cmd/tegola/cmd/cache.go

```diff
@@ -279,35 +167,234 @@ var cacheCmd = &cobra.Command{
 
 type MapTile struct {
 	MapName string
-	Tile    *tegola.Tile
+	Tile    *slippy.Tile
 }
 
-//parseTileString converts a Z/X/Y formatted string into a tegola tile
-func parseTileString(str string) (*tegola.Tile, error) {
-	var tile *tegola.Tile
+// parseTileString converts a Z/X/Y formatted string into a tegola tile
+// format string "{delim}{order}" (ex. "/zxy", " zxy", ",zxy"
+func parseTileString(format, str string) (*slippy.Tile, error) {
+	var tile *slippy.Tile
+
+	ix := 1
+	iy := strings.Index(format, "y") - 1
+	iz := strings.Index(format, "z") - 1
 
-	parts := strings.Split(str, "/")
+	parts := strings.Split(str, format[:1])
+	//parts := strings.Split(str, "/")
 	if len(parts) != 3 {
 		return tile, fmt.Errorf("invalid zxy value (%v). expecting the format z/x/y", str)
```

Comment:
We should return the correct format here.
Here is a play that should allow for parsing the format:
https://play.golang.org/p/HB3kz6n1End
assuming the above play:
```
return tile, fmt.Errorf("invalid zxy value(%v). expecting the format %v", str, format)
```

(As indicated above I would move most of the parsing into the format struct as well).